### PR TITLE
1590: Rename Environments

### PR DESCRIPTION
**Pull request by shaunharrisonFR**
Rename environments and sort out TTD Dev-cdk and devops mistake (assume not in use as not broken anything)

Issue: https://github.com/SecureApiGateway/SecureApiGateway/issues/1590

### CODE REVIEW

File: test-trusted-directory/kustomize/overlay/dev-aic/kustomization.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-namespace: devops-core
+namespace: dev-aic-core
```
Comment (wmorrison365fr) [inline test-trusted-directory/kustomize/overlay/dev-aic/kustomization.yaml:1]: git compare confused again?

File: ob/kustomize/overlay/dev-cdk/kustomization.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-namespace: nightly-ob
+namespace: dev-cdk-ob
```
Comment (wmorrison365fr) [inline ob/kustomize/overlay/dev-cdk/kustomization.yaml:1]: git compare confused again?

File: test-trusted-directory/kustomize/overlay/dev-cdk/kustomization.yaml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-namespace: nightly-core
+namespace: dev-cdk-core
```
Comment (wmorrison365fr) [inline test-trusted-directory/kustomize/overlay/dev-cdk/kustomization.yaml:1]: and again...
git compare confused again?

File: test-trusted-directory/kustomize/overlay/dev-aic/kustomization.yaml (hunk 1 above)
Comment (shaunharrisonFR) [reply test-trusted-directory/kustomize/overlay/dev-aic/kustomization.yaml:1]: No this one was wrong, it shouldnt have been devops but dev - but I don't think this branch is in use for the now dev-cdk environment, it uses master branch, which is why i've only now spotted this

(Master branch is used for envs that run v3, and ob-v4 branch is used when v4 is required)

File: ob/kustomize/overlay/dev-cdk/kustomization.yaml (hunk 2 above)
Comment (shaunharrisonFR) [reply ob/kustomize/overlay/dev-cdk/kustomization.yaml:1]: Yup!

File: test-trusted-directory/kustomize/overlay/dev-cdk/kustomization.yaml (hunk 3 above)
Comment (shaunharrisonFR) [reply test-trusted-directory/kustomize/overlay/dev-cdk/kustomization.yaml:1]: And Yup!

**Pull request closed without merging**